src/README.md

```diff
@@ -0,0 +1,27 @@
+## Welcome to the Babel Codebase
+
+Babel is broken down into three parts:
+
+- Parsing
+- Transformation
+- Generation
+
+**Parsing** is the process of turning a piece of code into an AST (Abstract
+Syntax Tree) which is a tree-like object of nodes that _describes_ the code.
+This makes it easier to transform the code over direct string manpulation.
+
+**Transformation** is the process of taking an AST and manipulating it into
+a new one. For example, Babel might take an ES2015 ArrowFunction and transform
```

Comment:
Maybe you can add something about how most of babel is in the transformation step like in

![image](https://cloud.githubusercontent.com/assets/588473/8468196/fb2af218-2036-11e5-8b8f-c7fec8e148bc.png)